README.md

```diff
@@ -19,6 +19,15 @@ By using the "Authenticate a new DID" part of the UI or posting the appropriate
 
 In the `Domain` Ledger screen ([example](http://greenlight.bcovrin.vonx.io/browse/domain)), you can browse through all of the transactions that have been created on this instance of the Ledger.  As well, you can use a drop down filter to see only specific Ledger transaction types (`nym` - aka DID, `schema`, `CredDef`, etc.), and search for strings in the content of the transactions.
 
+## Indy-Cli Container Environment
+
+This repository also provides a fully containerized Indy-Cli enviroment which allows you to use the Indy-Cli without having to build or install the Indy-SDK or any of it's dependancies on your machine.
+
+The environemnt provides a set of batch script templates and a simple variable substitution layer that allows the scripts to be reused for a number of purposes.
```

Comment:
Minor typos, 'enviroment', 'environemnt' a couple times, and 'dependancies'

Reply:
Fixed